sbin/build.sh

```diff
@@ -2221,7 +2221,7 @@ addJ9Tag() {
   # This code makes sure that a version number is always present in the release file i.e. openj9-0.21.0
   local j9Location="${BUILD_CONFIG[WORKSPACE_DIR]}/${BUILD_CONFIG[WORKING_DIR]}/${BUILD_CONFIG[OPENJDK_SOURCE_DIR]}/openj9"
   # Pull the tag associated with the J9 commit being used
-  J9_TAG=$(git -C $j9Location describe --abbrev=0)
+  J9_TAG=$(cd "$j9Location describe" && git --abbrev=0)
```

Comment:
This looks wrong - the `describe` shouldn't be part of the path
```suggestion
  J9_TAG=$(cd "$j9Location" describe && git describe --abbrev=0)
```

Reply:
@sxa oo, good spot! over zealous there